### PR TITLE
feat(jenkins): Adds csrf crumb config and handling.

### PR DESCRIPTION
**Pull request by jtk54**

### CODE REVIEW

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/model/Crumb.groovy [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.netflix.spinnaker.igor.model
```
Comment (lwander) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/model/Crumb.groovy]: Missing license header

**Comment by jtk54** [comment]
https://github.com/spinnaker/spinnaker/issues/2067.

**Comment by jtk54** [comment]
@tomaslin merging if no complaints.

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/JenkinsProperties.groovy [hunk 2]
```diff
@@ -45,6 +45,8 @@ class JenkinsProperties {
         @NotEmpty
         String password
 
+        Boolean preventCSRF = false
```
Comment (tomaslin) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/JenkinsProperties.groovy]: can we just change this to `csrf` and have it be true when it is enabled and false when not? `Cross-Site Request Forgery = true / false` is more user friendly.

**Comment by tomaslin** [comment]
LGTM

**Pull request merged**